presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/ActualProperties.java

```diff
@@ -298,18 +298,12 @@ private Global(Optional<Partitioning> nodePartitioning, Optional<Partitioning> s
 
         public static Global coordinatorSingleStreamPartition()
         {
-            return partitionedOn(
-                    COORDINATOR_DISTRIBUTION,
-                    ImmutableList.of(),
```

Comment:
the property of empty partitioning columns for single partitions needs to be maintained, otherwise we will lose quite a bit of inference power